Issue206 mixing volume

mwetter opened the pull request:
This will close #206

Mathadon commented:
In my opinion it would be better to leave out the resistances and set ```massDynamics``` to ```SteadyState```. This also solves the problem and it does not add complexity to the problem. Therefore it can serve as an example for users that encounter a similar problem.

Mathadon commented on Annex60/Fluid/MixingVolumes/Examples/MixingVolumeMFactor.mo:
```diff
@@ -73,6 +93,13 @@ difference between using an <code>mSenFac = 10</code> and multiplying volume by
 <li>
+March 26, 2015 by Michael Wetter:<br/>
+Added resistance between the volume and the pressure boundary
+condition to avoid an overspecified by consistent initial value problem.
```
'by' can be removed in this sentence

mwetter replied:
Yes, it should be "but consistent". I will fix it.

Mathadon commented on Annex60/Fluid/MixingVolumes/Examples/MixingVolumeMassFlow.mo:
```diff
@@ -45,6 +59,13 @@ This model demonstrates the use of the mixing volume with air flowing into and o
 <li>
+March 26, 2015 by Michael Wetter:<br/>
+Added resistance between the volume and the pressure boundary
+condition to avoid an overspecified by consistent initial value problem.
```
'by' can be removed

mwetter commented:
@Mathadon 
I agree to your proposed changes. Can you please review again. I did the following:

- Removed the pressure drops and changed the parameter for the mass dynamics to use unspecified initial conditions.
- Updated the info section to explain the rationale for this selection.
- Updated revision section
- Used in `MixingVolumeMFactor` the same instance for the boundary condition for all three volumes as I do not see why a second instance should be used.
- Uploaded new reference results.

Mathadon commented:
I actually proposed to use ```SteadyState``` massDynamics instead of ```DynamiceFreeInitial```, but if this works too then it's ok for me!
